bloom/generators/debian/generator.py

```diff
@@ -884,7 +884,8 @@ def generate_debian(self, package, debian_distro):
         # Template files
         template_files = process_template_files('.', subs)
         # Remove any residual template files
-        execute_command('git rm -rf ' + ' '.join(template_files))
+        execute_command('git rm -rf ' +
+            ' '.join("'%s'".format(t) for t in template_files))
```

Comment:
Halfway through this PR I tried using generator expressions rather than list comprehensions. I *think* the generator expression will work here in which case I'll propagate the change.